projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.scss

```diff
@@ -4,9 +4,8 @@
   .labs-popular-tags {
     .labs-popular-tag {
       color: $gunmetal;
-      font-size: $fs-h3;
       font-weight: $fw-400;
-      text-transform: lowercase;
+      @apply com-text-xl com-capitalize;
```

Comment:
Why have we changed from lowercase to capitalize?

Reply:
lowercase does not look good that's why!